include/pisa/compress.hpp

```diff
@@ -87,8 +87,8 @@ void compress_index(
 
         std::unique_ptr<index_scorer<WandType>> scorer;
 
-        if (scorer_name) {
-            scorer = scorer::from_name(*scorer_name, wdata);
+        if (quantized) {
```

Comment:
@elshize Attention here: Instead of passing an `optional` I have opted to pass the ScorerParam struct and use the `quantized` flag to check whether we need a ranker.